.circleci/config.yml

```diff
@@ -17,6 +17,14 @@ parameters:
     type: string
     default: 14.17-browsers
 
+executors:
+  integration-test:
```

Comment:
This basically copys what is in the named executor `hmpps/node_redis` (starting a node and redis container), then also starts the clamav container.
I couldnt work out how 'extend' an executor, or use 2 executors for a job ... 🤷‍♂️

Reply:
No no-one does! I asked about this the other week! I think we need some way of having a single executor where you can select which images to include... but the godfather of orbs left a while back.